app/src/main/java/org/mozilla/tv/firefox/fxa/FxaRepo.kt

```diff
@@ -81,10 +79,11 @@ class FxaRepo(
     private val _accountState: BehaviorSubject<AccountState> = BehaviorSubject.createDefault(NotAuthenticated)
     val accountState: Observable<AccountState> = _accountState.hide()
 
-    val receivedTabs: Observable<ReceivedTabs> = admIntegration.receivedTabsRaw
-        .mapToReceivedTabs()
-        .filterInvalidTabs(sentryIntegration)
-        .doOnNext { telemetryIntegration.receivedTabEvent(it) }
+    val receivedTabs: Observable<Consumable<FxaReceivedTab>> = admIntegration.receivedTabsRaw
+        .filterMapToDomainObject()
+        .map { Consumable.from(it) }
```

Comment:
nit: What this Consumable -> replay -> autoConnect pattern is doing is opaque, at least if you're unfamiliar with Rx. Should we add a comment to explain?

fwiw, because it's so opaque, it could be a good area to test so devs don't accidentally change this later because we're relying on it to make send tab function the way we want it to. The test I imagine would be from the perspective of the `receivedTabs` consumer (i.e. the application), ensuring it'll replay events after subscribing, unsubscribing, resubscribing and things like that.